### ISSUE TITLE
Bad docker after 2021.1

### ISSUE THREAD

**Issue by vtimofeev01** [intel/video-analytics-serving#37]
I try to make a new image an use gvatrack there
if I use `./build` - the gvatrack is looking for  libopencv_video.so.4.4 and does not start
if I use build `./docker/build.sh --framework gstreamer --base openvino/ubuntu18_data_dev` I cant start vaserving within the container
if I use `./docker/build.sh --framework gstreamer --base openvisualcloud/xeone3-ubuntu1804-analytics-gst` gvatrack looks for 
```
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...
ERROR: from element /GstPipeline:pipeline0/GstGvaTrack:gvatrack0: tracker intitialization failed
Additional debug info:
/home/gst-video-analytics/gst/elements/gvatrack/gstgvatrack.c(159): gst_gva_track_set_caps (): /GstPipeline:pipeline0/GstGvaTrack:gvatrack0:

dlopen() failed: libopencv_video.so.4.3: cannot open shared object file: No such file or directory

ERROR: pipeline doesn't want to preroll.
Setting pipeline to NULL ...
Freeing pipeline ...
```
please tell the right way

**Comment by nnshah1** [comment]
@vtimofeev01 

We are in the process of preparing an update which should allow use of `openvino/ubuntu18_data_dev` and `openvino/ubuntu18_runtime`. We will provide instructions by end of week. Either here as a patch or as a preview release. Sorry for delay. 

In the meantime you can also try:

`--base openvino/ubuntu18_data_dev:2020.4` instead of 'latest'

**Comment by whbruce** [comment]
Fixed in [v0.4.0-alpha-preview](37a10aafa772218bd9ac2d69ebeabc99039a24aa)

**Comment by nnshah1** [comment]
@vtimofeev01 Wanted to confirm that you were able to get `openvino/ubuntu18_runtime:2021.1` working with `gvatrack` enabled
